beeflow/client/core.py

```diff
@@ -382,6 +382,57 @@ def stop():
     print(f'Beeflow has stopped. Check the log at "{beeflow_log}".')
 
 
+@app.command()
+def reset(archive: bool = typer.Option(False, '--archive', '-a',
+                                       help='Archive .beeflow before removal')):
+    """Delete the .beeflow directory."""
+    # Check to see if the user is absolutely sure. Warning Message.
+    absolutely_sure = ""
+    while absolutely_sure != "y" or absolutely_sure != "n":
+        # Get the user's .beeflow directory
+        directory_to_delete = os.path.expanduser("~/.beeflow")
```

Comment:
The directory should not be hard coded. We use bee_workdir in the bee configuration file for the directory. The default is ~/.beeflow but the user could actually change that to some other location.

Reply:
Good catch. I didn't realize that was one of the config options.